.gitignore

```diff
@@ -6,4 +6,4 @@ DerivedData/
 .swiftpm/configuration/registries.json
 .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata
 .netrc
-rust/bevy_ios_notifications/Cargo.lock
+/bevy_ios_safearea/Cargo.lock
```

Comment:
not needed. there is a gitignore right in the subfolder

Reply:
Done